SPECS/numpy/numpy.spec

```
@@ -1,131 +1,857 @@
+arrays.  NumPy is built on the Numeric code base and adds features
+introduced by numarray as well as an extended C-API and the ability to
+create arrays of arbitrary type.
+
+There are also basic facilities for discrete fourier transform,
+basic linear algebra and random number generation. Also included in
+this package is a version of f2py that works properly with NumPy.
+
+%package -n python3-numpy
+%{?python_provide:%python_provide python3-numpy}
+Summary:        A fast multidimensional array facility for Python
+License:        BSD
+BuildRequires:  %{blaslib}-devel
+BuildRequires:  chrpath
+BuildRequires:  gcc
 BuildRequires:  gcc-c++
 BuildRequires:  gcc-gfortran
 BuildRequires:  lapack-devel
+BuildRequires:  python3-Cython
 BuildRequires:  python3-devel
-BuildRequires:  python3-setuptools
-BuildRequires:  python3-Cython >= 0.29.24
+BuildRequires:  python3-setuptools < 60
+Provides:       libnpymath-static = %{version}-%{release}
+Provides:       libnpymath-static%{?_isa} = %{version}-%{release}
+Provides:       numpy = %{version}-%{release}
+Provides:       numpy%{?_isa} = %{version}-%{release}
+Obsoletes:      numpy < 1.10.1-3
 %if %{with_check}
 BuildRequires:  python3-pip
 %endif
 
-%description
-NumPy is a general-purpose array-processing package designed to efficiently manipulate large multi-dimensional arrays of arbitrary records without sacrificing too much speed for small multi-dimensional arrays. NumPy is built on the Numeric code base and adds features introduced by numarray as well as an extended C-API and the ability to create arrays of arbitrary type which also makes NumPy suitable for interfacing with general-purpose data-base applications.
-
-%package -n     python3-numpy
-Summary:        python-numpy
-Requires:       python3
-
 %description -n python3-numpy
-Python 3 version.
+NumPy is a general-purpose array-processing package designed to
+efficiently manipulate large multi-dimensional arrays of arbitrary
+records without sacrificing too much speed for small multi-dimensional
+arrays.  NumPy is built on the Numeric code base and adds features
+introduced by numarray as well as an extended C-API and the ability to
+create arrays of arbitrary type.
+
+There are also basic facilities for discrete fourier transform,
+basic linear algebra and random number generation. Also included in
+this package is a version of f2py that works properly with NumPy.
 
 %package -n python3-numpy-f2py
+%{?python_provide:%python_provide python3-numpy-f2py}
 Summary:        f2py for numpy
+License:        BSD AND Python AND ASL 2.0
 Requires:       python3-devel
-Requires:       python3-numpy = %{version}-%{release}
+Requires:       python3-numpy%{?_isa} = %{version}-%{release}
 Provides:       python3-f2py = %{version}-%{release}
+Obsoletes:      python3-f2py <= 2.45.241_1927
+Provides:       f2py = %{version}-%{release}
+Provides:       numpy-f2py = %{version}-%{release}
+Obsoletes:      numpy-f2py < 1:1.10.1-3
```

Also not needed `Obsoletes` - we never had these packages anyway.

A bit of Mariner trivia: what's probably more important, we've "reset" the epoch numbers on almost all of our packages, so `numpy-f2py < 1:1.10.1-3` would be even problematic, if we ever introduced it as a separate package.

```suggestion
Provides:       f2py = %{version}-%{release}
Provides:       numpy-f2py = %{version}-%{release}
```